Parent/Parent/Dashboard/DashboardViewController.swift

```diff
@@ -272,12 +274,24 @@ class DashboardViewController: UIViewController, CustomNavbarProtocol {
                 self?.badgeCount = unreadCount
                 performUIUpdate {
                     let color = ColorScheme.observee(currentStudentID ?? "0").color
-                    self?.navigationItem.leftBarButtonItem?.addBadge(number: unreadCount, color: color)
+                    self?.updateBadgeCount(color: color)
                 }
             } }
         }
     }
 
+    func updateBadgeCount(color: UIColor) {
+        navigationItem.leftBarButtonItem?.addBadge(number: badgeCount, color: color)
+        let accessibilityLabel: String
+        if badgeCount > 0 {
+            let template = NSLocalizedString("Settings. %d unread conversations.", comment: "")
```

Comment:
I know Parent doesn't have a `stringdict` file yet, but I think it would be ideal to add this to one because we will need to change `conversations` to be singular when the value is 1.

Reply:
good catch.  fixed